### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by Siedlerchr**
German translation updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Automatically generated group names are now converted from LaTeX to Unicode
 - Unified dialogs for opening/saving files
 - Add integrity check to avoid non-ASCII characters in BibTeX files
+- Updated German translation
```
Comment (oscargus) [inline CHANGELOG.md]: I do not see any need for this. Or rather, no other languages state the updates. (I think French, Turkish, and Swedish are updated recently as well.) Maybe we should collect this in a single point before each release?

File: src/main/resources/l10n/JabRef_de.properties [hunk 2]
```diff
@@ -2448,17 +2448,17 @@ Shared_version\:_%0=Gemainsam_genutzte_Version\:_%0
...
 The_BibEntry_you_currently_work_on_has_been_deleted_on_the_shared_side._Hit_"Keep"_to_recover_the_entry.=Der_Eintrag,_welchen_Sie_zur_Zeit_bearbeiten,_ist_auf_der_gemeinsam_genutzen_Datenbank_nicht_mehr_vorhanden._Klicken_Sie_auf_"Behalten",_um_den_Eintrag_wiederherzustellen.
-Cannot_cite_entries_without_BibTeX_keys._Generate_keys_now?=
+Cannot_cite_entries_without_BibTeX_keys._Generate_keys_now?=Kann_keine_Einträge_ohne_BibTeX-Key_zitieren._Keys_jetzt_generieren?
 New_technical_report=Neuer_technischer_Bericht
 
-%0_file=
-Custom_layout_file=
-Protected_terms_file=
-Style_file=
+%0_file=%0_-Datei
```
Comment (oscargus) [inline src/main/resources/l10n/JabRef_de.properties]: Keine _ hier?

File: CHANGELOG.md (hunk 1 above)
Comment (koppor) [inline CHANGELOG.md]: Yes, no CHANGELOG.md entry for language updates

File: src/main/resources/l10n/JabRef_de.properties (hunk 2 above)
Comment (koppor) [inline src/main/resources/l10n/JabRef_de.properties]: I see the `_`.

Comment (Siedlerchr) [inline src/main/resources/l10n/JabRef_de.properties]: Yes, It is `+%0_file=%0_-Datei`

Comment (oscargus) [inline src/main/resources/l10n/JabRef_de.properties]: OK. So PubMed -Datei? German is a weird language. ;-)

Comment (grimes2) [inline src/main/resources/l10n/JabRef_de.properties]: In my German version of JabRef, I see translations like ArXiv-Spalte, BibTeX-Felder and BibTeX-Datei. To be consistent it should be PubMed-Datei and not PubMed -Datei.

Comment (koppor) [inline src/main/resources/l10n/JabRef_de.properties]: @oscargus You are right in spotting the issue. It has to be **without** the space. I think, both @Siedlerchr and me did not get your comment. @grimes2: Thank you for confirming.

I just fixed it in the master branch. Thank you for spotting.

**Pull request merged**